test/test_extended_xsort.cpp

```diff
@@ -105,8 +106,8 @@ namespace xt
         check_argpartition_equal(py_a, py_a19, xt::argpartition(py_a, 19), 19);
 
         // py_median = np.median(a)
-        long py_median = 300;
-        EXPECT_EQ(py_median, xt::median(py_a));
+        double py_median = 300.0;
+        EXPECT_EQ(static_cast<long>(py_median), xt::median(py_a));
```

Comment:
Something is strange here...

Reply:
Thanks for the review @tdegeus ! I have the same feeling. But my intention is just to reproduce the existing tests :-)

Reply:
Maybe a bit more explanation! Otherwise it could be confusing.

In the new file, ```double py_median = 300.0;``` is automatically generated.
In the old file, ```long py_median = 300;``` was manually modified. In order to maintain the old test, although I have no idea why, I added the cast.

Reply:
Got it. Let's keep it as you propose then. Unless you want to consider `static_cast<decltype(py_a)::value_type>`

Reply:
Good point! I implemented your suggestion and also added a comment (to make people aware what's happened in the past).